runtime/lua/vim/lsp.lua

```diff
@@ -103,10 +103,11 @@ end
 
 local wait_result_reason = { [-1] = "timeout"; [-2] = "interrupted"; [-3] = "error" }
 
+local UTF8, UTF16,  UTF32 = protocol.Encodings.UTF8, protocol.Encodings.UTF16, protocol.Encodings.UTF32
```

Comment:
there's a multi-space gap, we don't use in general multiple declaring/unpacking locals on the same line.